### ISSUE TITLE
Errors like "[bxerr:BASX0000] java.lang.IllegalArgumentException: Points of LinearRing do not form a closed linestring"

### ISSUE THREAD

**Issue by cportele** [inspire-eu-validation/ets-repository#41]
If the GML import in the deegree library used by GmlGeoX cannot parse a geometry due to errors, it throws an exception. All ggeo:getGeometry() functions should be protected with try/catch blocks like

```
	try {
		...
	} catch * { 
		let $dummy := local:log($err:code || ' - ' || $err:description || ' - ' || $err:module || ' - ' || $err:line-number || ' - ' || $err:column-number || ' - ' || $err:additional)
		return () (: parsing exceptions already reported by gmlas.d.11 :)
	}
```

**Comment by jonherrmann** [comment]
Duplicate #44

**Comment by cportele** [comment]
@jonherrmann - this is not a duplicate of that issue

**Comment by jonherrmann** [comment]
OK, I see the difference now. But this kind of errors will be caught and reported as well, Clemens?

**Comment by cportele** [comment]
Yes, but only once in the basic tests.
